core/src/main/java/org/dozer/config/resolvers/SystemEnvironmentSettingsResolver.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2005-2017 Dozer Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dozer.config.resolvers;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Settings resolver which are system environment variables
+ */
+public class SystemEnvironmentSettingsResolver implements SettingsResolver {
+
+    private final SettingsResolver fallthrough;
+
+    public SystemEnvironmentSettingsResolver(SettingsResolver fallthrough) {
+        this.fallthrough = fallthrough;
+    }
+
+    @Override
+    public void init() {
+        //noop
```

Comment:
In some implementations we do `fallthrough.init();`, in other do not. Maybe we should follow the same approach everywhere? 
Also in my understanding `fallthrough` is optional, so in places we do init it, we should probably check for `null`.